scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -411,12 +563,16 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
                     .then(function(value) {
                         notify.success(gettext('Item sent.'));
 
-                        // Remember last destination desk and stage
-                        if (scope.destination_last &&
-                                (scope.destination_last.desk !== deskId && scope.destination_last.stage !== stageId)) {
-                            updateLastDestination(deskId, stageId);
-                        } else {
-                            updateLastDestination(deskId, stageId);
+                        if (scope.currentUserAction === scope.userActions.send_to) {
+                            // Remember last destination desk and stage for send_to.
+                            var last_destination = scope.destination_last[scope.currentUserAction];
+                            var key = PREFERENCE_KEY;
+
+                            if (!last_destination) {
```

Comment:
The outcome of both these conditions is exactly the same. You can simplify this.